src/Umbraco.Cms.Integrations.Search.Algolia/Umbraco.Cms.Integrations.Search.Algolia.csproj

```diff
@@ -17,6 +17,7 @@
 		<Authors>Umbraco HQ</Authors>
 		<Company>Umbraco</Company>
 		<PackageTags>Umbraco;Umbraco-Marketplace</PackageTags>
+		<PackageIcon>https://github.com/umbraco/Umbraco.Cms.Integrations/tree/main/src/Umbraco.Cms.Integrations.Search.Algolia/logo/algolia.png</PackageIcon>
```

Comment:
You should check if this actually resolves to the image file.  You might need actually to click on the "Raw" link when you are viewing this page on GitHub, and use that as the URL.